src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -42,6 +45,12 @@ public Command parseCommand(String userInput) throws ParseException {
 
         final String commandWord = matcher.group("commandWord");
         final String arguments = matcher.group("arguments");
+
+        // Note to developers: Change the log level in config.json to enable lower level (i.e., FINE and lower)
+        // log messages such as the one below.
+        // Currently, lower level log messages are used sparingly, to minimize noise in the code.
+        logger.fine("Command word: " + commandWord + " ; Arguments: " + arguments);
```

Comment:
Perhaps this comment can be rephrased so it doesn't describe a point in time, as markers like "currently" make code comments harder to maintain.

Instead, it could describe the desired usage, for e.g. "Lower level log messages should be used sparingly to minimise noise in the code."

Also there's a space before the inline semicolon, not sure if that's intended.

```suggestion
        // Lower level log messages should be used sparingly to minimise noise in the code..
        logger.fine("Command word: " + commandWord + "; Arguments: " + arguments);
```

Reply:
> Instead, it could describe the desired usage, for e.g. "Lower level log messages should be used sparingly to minimise noise in the code."

Good point @hansstanley